apps/backend/src/lib/outbox-listener.ts

```diff
@@ -34,6 +36,10 @@ export class OutboxListener {
     this.fallbackPollMs = options.fallbackPollMs ?? 500
   }
 
+  setCompanionService(service: CompanionService): void {
```

Comment:
Is there a reason for a setting instead of params? I don't think we would want the companion service to be optional either, so why not just pass it in directly?

If you're thinking of the e2e tests, we should use a stub similar to auth if we don't actually want the AI there (which we don't).